csharp/ScheduledEventsSample/ScheduledEventApproval.cs

```diff
@@ -16,21 +16,28 @@
 namespace ScheduledEventsSample
 {
     using System.Collections.Generic;
+    using System.Runtime.Serialization;
     using Newtonsoft.Json;
 
+    /// <summary>
+    /// Represents the document structure used to expedite scheduled events
+    /// </summary>
     public class ScheduledEventsApproval
     {
         public List<StartRequest> StartRequests = new List<StartRequest>();
     }
 
+    /// <summary>
+    /// An individual request to expedite an event
+    /// </summary>
     public class StartRequest
```

Comment:
I think in order to fully support DataMembers, the class needs to be marked with the [DataContract attribute](https://msdn.microsoft.com/en-us/library/system.runtime.serialization.datacontractattribute(v=vs.110).aspx)